landscape.yml

```diff
@@ -1800,6 +1800,19 @@ landscape:
               accepted: '2021-04-28'
               dev_stats_url: https://wasmedge.devstats.cncf.io/
               artwork_url: https://github.com/cncf/artwork/blob/master/examples/sandbox.md#wasmedgeruntime-logos
+          - item:
```

Comment:
( I was about to create a new subcategory "Containers for Desktop Operating Systems", but doesn't seem allowed)
https://github.com/cncf/landscape#new-entries )

> We are unlikely to create a new category for products/projects as we'd rather find the best home with the current options.